portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm

```diff
@@ -169,6 +171,18 @@
 
             document.querySelector('.Mrphs-portalWrapper').focus();
         }, 50);
+
+        const siteDescPopoverList = Array.prototype.slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'));
```

Comment:
You can just use spread here. It does the same thing and is more concise. In fact, if you use spread, you can probably get rid of the variable and have the whole thing on one line.
[...document.querySelectorAll('[data-bs-toggle="popover"]')].forEach ...